### ISSUE TITLE
Generate a string with specified separators

### ISSUE THREAD

**Issue by php-coder** [qala-io/datagen#16]
I need to generate a string that could consist from words. I'm trying the following:
```java
length(10).with(oneOf(" ,")).english()
```
but sometimes it generates strings that start or end with space or comma. I'm looking to a way of avoiding it. Would it be possible to also specify a separator between words?

**Comment by ctapobep** [comment]
Sounds useful. Something like this: `length(10).with(wordSeparators()).english()`?

**Comment by php-coder** [comment]
Yes. Could be even shorter: `length(10).with(separator(" ,")).english()`

**Comment by ctapobep** [comment]
Ok, I don't know if I implement this in the nearest future. So in the meantime if this is acceptable, you can try repeaters: `repeat(length(10), NUMERIC).string(", ").times(4)`
